src/vcpkg/vcpkgcmdarguments.cpp

```diff
@@ -290,6 +316,18 @@ namespace vcpkg
                                  StabilityTag::Experimental,
                                  args.asset_sources_template_arg,
                                  msg::format(msgAssetSourcesArg));
+        {
+            std::string raw_cmake_debug;
+            if (args.parser.parse_option(CMAKE_DEBUGGING_ARG, StabilityTag::Experimental, raw_cmake_debug))
```

Comment:
I wonder if `parse_option` couldn't be thought to do this automatically if fulfills the concept of being StringConstructable.

Reply:
How is this:
https://learn.microsoft.com/en-us/vcpkg/commands/common-options#asset-sources
https://learn.microsoft.com/en-us/vcpkg/commands/common-options#binarysource
getting parsed?

Aren't those options using a kind of comma separation?